app/components/dataset-file-grid/component.js

```diff
@@ -2,9 +2,14 @@ import Ember from 'ember';
 import pagedArray from 'ember-cli-pagination/computed/paged-array';
 
 export default Ember.Component.extend({
+
     store: Ember.inject.service(),
     session: Ember.inject.service(),
-    
+    router: Ember.inject.service('-routing'),
+
+    classNames: ["item-details"],
```

Comment:
these class names for each section aren't standardized. This one is "item-details", another one is "items", another one is "landing-paragraph-container". They should all be named according to the same convention